src/elife_profile/modules/custom/elife_person_profile/css/person-profile.css

```diff
@@ -100,15 +100,27 @@
     font-size: 2rem;
 }
 
-.person-profile__biog-expertise {
+.person-profile__biog-expertise, .person-profile__biog-keywords {
     margin: 1em 0;
 }
 
-.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-expertise ul {
+.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-expertise ul, .person-profile__biog-wrapper .person-profile__biog .person-profile__biog-keywords ul {
     padding-left: 0;
     padding-right: 0;
 }
 
+.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-keywords ul li {
```

Comment:
This selector chain and those below have at least 5 levels, which is very deep. Are all these levels required to generate high enough specificity for the rule to apply, or could they be shortened and still give the desired result?